update

KCMinchakpu opened the pull request:
Added the Seller's Profile codes

KCMinchakpu committed:
update

Added the Seller's Profile codes

crewsbd commented:
There is an incompatibility with the package manager. The project manager is npm, but pnpm was used on this branch. Try to change it to npm.

KCMinchakpu committed:
Update package-lock.json

update

crewsbd commented:
We need to remove the pnpm.yaml stuff. Just stick with npm

KCMinchakpu committed:
update

switched pnpm to npm and deleted the pnpm-lock.yaml file from the project directory

KCMinchakpu committed:
update

Renamed seller-profile.tsx to page.tsx

KCMinchakpu committed:
Update page.tsx

just imported React from react

KCMinchakpu committed:
update

just imported React into scope in some pages

KCMinchakpu committed:
update

Pull the main branch into my branch and  Resolved conflicts

crewsbd committed:
Adjusting package.json

KCMinchakpu committed:
update

Removed useDropzone, added an <input type="file" /> for uploading images, and Handled image selection using handleImageUpload to update the productImages state and to be able to Display the uploaded images after selection.

KCMinchakpu committed:
Merge branch 'sellers' of https://github.com/HandcraftedHaven-Group09/handcrafted-haven into sellers

KCMinchakpu committed:
Update

Removed useDropzone and added an <input type="file" /> for uploading images.

KCMinchakpu committed:
Merge branch 'main' into sellers

crewsbd committed:
Minor changes
